packages/integration-karma/test/api/CustomElementConstructor-getter/x/lifecycleChild/lifecycleChild.js

```diff
@@ -3,18 +3,26 @@ import { LightningElement } from 'lwc';
 export default class Child extends LightningElement {
     constructor() {
         super();
-        window.timingBuffer.push('child:constructor');
+        if (window.timingBuffer) {
+            window.timingBuffer.push('child:constructor');
+        }
     }
 
     connectedCallback() {
-        window.timingBuffer.push('child:connectedCallback');
+        if (window.timingBuffer) {
+            window.timingBuffer.push('child:connectedCallback');
+        }
     }
 
     disconnectedCallback() {
-        window.timingBuffer.push('child:disconnectedCallback');
+        if (window.timingBuffer) {
+            window.timingBuffer.push('child:disconnectedCallback');
+        }
```

Comment:
The element might be disconnected from the DOM in the `afterEach()`, at which point `window.timingBuffer` has been removed by the test.